components/fl-operator/pkg/resources/resources.go

```diff
@@ -66,11 +67,16 @@ func NewPod(task *pb.OrchestratorMessage_TaskSpec, name types.NamespacedName, en
 			Labels:    labels,
 		},
 		Spec: corev1.PodSpec{
+			ShareProcessNamespace: &shareProcessNamespace,
```

Comment:
Please add a comment why `ShareProcessNamespace = true` is necessary here.

Reply:
Good point. It's a good way to clarify the intention.